app/src/test/kotlin/compiler/e2e/ParserErrorsTest.kt

```diff
@@ -1,5 +1,441 @@
+                        """,
+            )
+        )
+    }
+
+    @Test
+    fun `test expressions`() {
+        assertParseError(
+            listOf(
+                """
+                            czynność f(x: Liczba) -> Liczba {
+                                zwróć x
+                                + x
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Liczba) -> Liczba {
+                                zwróć (x %;x)
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Liczba) -> Liczba {
+                                zwróć ()
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Liczba) -> Liczba {
+                                zwróć (x - x
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Czy) -> Czy {
+                                zwróć )x albo x(
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Czy) -> Czy {
+                                zwróć [nie x]
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Liczba) -> Czy {
+                                zwróć {x >= x}
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Liczba) -> Czy {
+                                zwróć (x !== x)
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Liczba) -> Czy {
+                                zwróć (x === x)
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Czy) -> Czy {
+                                czynność g(y: Czy) -> Czy { }
+                                zwróć g(x: Czy)
+                            }
+                            
+                        """,
+            )
+        )
+    }
+
+    @Test
+    fun `test operators`() {
+        assertParseError(
+            listOf(
+                """
+                            czynność f(x: Liczba) {
+                                x
+                                ++
+                            }
+                            
+                        """,
+                """
+                            czynność f(x: Liczba) -> Liczba {
+                                --
+                                x
+                            }
```

Comment:
My bad once again, but it turns out we don't have increment/decrement operators, so we should remove this as well